bin/fake-exe

```diff
@@ -15,7 +15,7 @@
 set -u
 
 EXE=$1
-TMP='.tmp o'
```

Comment:
That was actually deliberate, to make the temporary files more obvious in a "watched" directory listing. However, it's not necessary and probably looks like a mistake!